src/screens/Profiles/UserLibrary/components/UserLibraryListCard/component.js

```diff
@@ -54,6 +54,23 @@ export class UserLibraryListCard extends React.Component {
     isSliderActive: false,
   }
 
+  componentWillReceiveProps(nextProps) {
+    const { libraryEntry, libraryType } = nextProps;
+    if (this.props.libraryEntry.id !== libraryEntry.id) {
+      const mediaData = libraryEntry[libraryType];
+      const maxProgress = mediaData.episodeCount || mediaData.chapterCount;
```

Comment:
should be `mediaData.episodeCount || mediaData.chapterCount || 1` since we're going to be dividing by this value.

Reply:
Yeah, the purpose is to hit a `NaN` since we don't display the progress bar if the `typeof` isn't number.

Reply:
We'd have to change that logic too 🤔

Reply:
ah alright i didn't know how the progress thing was displayed